spec/models/child_theme_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+describe ChildTheme do
+  it "doesn't allow a user selectable theme to be a child" do
+    parent = Theme.create!(name: "parent", user_id: -1)
```

Comment:
It seems like we added a fabricator for `theme`, would be better to use that instead.